making_change/making_change.py

```diff
@@ -3,7 +3,25 @@
 import sys
 
 def making_change(amount, denominations):
-  pass 
+  #if amount is less than 1 return 0
+  #if 
+
```

Comment:
You're off to a good start, these are tough.  My suggestion would to try and write out and talk through the process of the problem , it could help and benefit you in interviews 👍